src/library/bpmdelegate.cpp

```diff
@@ -18,10 +19,10 @@ class BpmEditorCreator : public QItemEditorCreatorBase {
     }
 
     QWidget* createWidget(QWidget* parent) const override {
-        QDoubleSpinBox* pBpmSpinbox = new QDoubleSpinBox(parent);
+        WDoubleSpinBox* pBpmSpinbox = new WDoubleSpinBox(parent);
         pBpmSpinbox->setFrame(false);
         pBpmSpinbox->setMinimum(0);
-        pBpmSpinbox->setMaximum(1000);
+        pBpmSpinbox->setMaximum(99999);
```

Comment:
Any reason for allowing this much BPM?

Reply:
Obviously sky is the limit ; )
https://github.com/mixxxdj/mixxx/issues/12942#issuecomment-1986566470